object-browser.md

```diff
@@ -0,0 +1,68 @@
+```hbs
+{{frost-object-browser-lts
+  items=sortedItems
+  header=(component 'frost-info-bar'
+    icon=(component 'frost-icon'
+      icon='list-medium'
+      pack='frost'
+    )
+    title=(component 'text-box'
+      isVisible=isTitleVisible
+      text='&lt;placeholder: title&gt;'
+    )
+    summary=(component 'text-box'
```

Comment:
I think if you check the latest info-bar this is either text or a custom component (i.e. `text-box` shouldn't be a thing)

If that's not the case we need to fix info-bar